app/helpers/component_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ComponentHelper
+  def add_option(option, output = nil)
```

Comment:
When commenting on a function like this, please do so at the end. Otherwise I have no clue what it is currently without switching to the other view.